src/main/java/com/sonyericsson/jenkins/plugins/bfa/ScanLogAction.java

```diff
@@ -20,6 +23,12 @@ public class ScanLogAction implements RunAction2 {
 
     private transient Run run;
 
+    private long startTime = System.currentTimeMillis();
+
+    private Long endTime;
+
+    private Exception exception;
```

Comment:
FWIW, exceptions are not always lightweight from a serialization standpoint. Maybe safer to just store `Exception.toString` as a `String exceptionMessage` or even `boolean scanFailed` and let people look at the log for details.

Reply:
See https://github.com/jenkinsci/build-failure-analyzer-plugin/pull/175/commits/aa846cfd31d12d4a4743aab405af584504ac769c